### ISSUE TITLE
Belongstomany

### ISSUE THREAD

**Issue by tgoeminne** [riesenia/cakephp-duplicatable#7]
It appears to not work with belongstomany.

For example contain tags. It will duplicate tags instead of the tags_articles table.

**Comment by davidyell** [comment]
I've found that it works perfectly for `belongsToMany`. Perhaps you have a bug in your table relationships, a typo, or haven't contained the right tables.

**Comment by segy** [comment]
@tgoeminne can you please provide an example? thanks

**Comment by ADmad** [comment]
The tests cases currently don't cover belongsTo and belongsToMany associations :)

**Comment by davidyell** [comment]
If it's any help @tgoeminne, here is how I used it in my application shell class. All of the contained Table classes are `belongsToMany`

```php
<?php
$contain = [];
if ($this->param('copy')) {
    $contain = [
        'Attributes',
        'Offers',
        'Packages',
        'Pricings',
        'Upsells'
    ];
}

$this->ApiUsers->behaviors()->load('Duplicatable.Duplicatable', [
    'contain' => $contain,
    'remove' => ['created', 'modified'],
    'set' => [
        'name' => $partnerName,
        'username' => $partnerUsername,
        'keycode' => $partnerKey,
        'partner_installation_id' => $partnerInstallId
    ],
]);

$this->ApiUsers->duplicate(6);
```

**Comment by segy** [comment]
I added test for belongstomany and @tgoeminne was right. fixed in 1.0.6
